registry.yaml

```diff
@@ -341,3 +341,27 @@ plugins:
         supported: true
         id: 999
         source: test
+  - name: k8smeta
+    description: Enriche Falco syscall flow with Kubernetes Metadata
+    authors: The Falco Authors
+    contact: https://falco.org/community
+    maintainers:
+      - name: The Falco Authors
+        email: cncf-falco-dev@lists.cncf.io
+    keywords:
+      - kubernetes
+      - syscall
+      - extractor
+    url: https://github.com/falcosecurity/plugins/tree/master/plugins/k8smeta
+    license: Apache-2.0
+    signature:
+      cosign:
+        certificate-oidc-issuer: https://token.actions.githubusercontent.com
+        certificate-identity-regexp: https://github.com/falcosecurity/plugins/
+    capabilities:
+      extraction:
+        supported: true
+      parsing:
+        supported: true
+      async:
+        supported: true
```

Comment:
not clear to me which are the supported values under `capabilities`, do we have a validation for this field?

Reply:
I guess we don't have it yet.

Reply:
We don't have it yet :point_down: 
https://github.com/falcosecurity/plugins/blob/master/build/registry/pkg/registry/types.go#L39

Could you remove `async` at the moment, please? We can still introduce it later.

Reply:
done I used just `extraction` and added the `sources` field. Thank you for the pointer!